js/src/ExpectedMark.js

```diff
@@ -31,6 +31,10 @@ export default class ExpectedMark extends UXBase {
 		return _expectedMarks[name];
 	}
 
+	static clearExpectedMarks() {
```

Comment:
I need this for unit tests. Might be a method we hook into in SPA as well.

Reply:
Currently this is just clearing the `_expectedMarks` cache for unit tests. If we use it in SPA (which I think we might) then yes we would also want it to clear from the PerformanceTimeline. However, unsure of that yet so I didn't add it in.